src/xenia/kernel/xboxkrnl/xboxkrnl_module.cc

```diff
@@ -189,6 +190,11 @@ XboxkrnlModule::XboxkrnlModule(Emulator* emulator, KernelState* kernel_state)
                                        ordinals::XexExecutableModuleHandle,
                                        ppXexExecutableModuleHandle);
 
+  uint32_t ppExLoadedImageName =
```

Comment:
Need a comment about 256 bytes (rather than 260 MAX_PATH) being always allocated by the original kernel, in their entirety, disregarding the actual length.